### PR TITLE
[17.0][MIG] account_payment_return_import: Migration to version 17.0

### PR DESCRIPTION
**Pull request by carolinafernandez-tecnativa**
-Standard migration to v17.

@Tecnativa
TT49283
@carlos-lopez-tecnativa @pedrobaeza

### CODE REVIEW

File: account_payment_return_import/tests/test_import_file.py [hunk 1]
```diff
@@ -22,16 +24,6 @@ class TestPaymentReturnFile(TransactionCase):
     @classmethod
     def setUpClass(cls):
```
Comment (celm1990) [inline account_payment_return_import/tests/test_import_file.py]: Remove the method if it is empty.

**Review by celm1990 — APPROVED** [review]
LGTM only a minor change

File: account_payment_return_import/tests/test_import_file.py (hunk 1 above)
Comment (carolinafernandez-tecnativa) [reply account_payment_return_import/tests/test_import_file.py]: Done, thanks :)

File: account_payment_return_import/__manifest__.py [hunk 2]
```diff
@@ -1,13 +1,14 @@
...
+# Copyright 2024 Tecnativa - Carolina Fernandez
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
 {
     "name": "Account Payment Return Import",
     "category": "Accounting",
-    "version": "16.0.1.0.1",
-    "development_status": "Mature",
+    "version": "17.0.1.0.0",
+    "development_status": "Production/Stable",
```
Comment (pedrobaeza) [inline account_payment_return_import/__manifest__.py]: I think we should put as Mature `account_payment_return` instead.

**Comment by pedrobaeza** [comment]
Please do it in a separate commit

**Comment by carolinafernandez-tecnativa** [comment]
ping @pedrobaeza

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**